pod/perlguts.pod

```diff
@@ -1889,6 +1889,20 @@ and back.
 =for apidoc Amh||SAVESPTR|SV * s
 =for apidoc Amh||SAVEPPTR|char * p
 
+=item C<SAVERCPV(char *pv)>
+
+This macro arranges to restore the value of a C<char *> which was
+allocated with a call to C<rcpv_new()> to its previous state when the
+current pseudo block is completed. The pointer stored in C<pv> at the
+time of the call will be refcount incremented and stored on the save
+stack. Later when the current pseudoblock is completed the value stored
+in C<&rcpv> will be refcount decremented, and the previous value
+restored from the savestack and also be refcount decremented.
+
+This is the C<RCPV> equivalent of C<SAVEFREESV()>.
```

Comment:
Did you mean to put this line in the docs for `SAVEFREERCPV()`? It doesn't seem to make sense here

Reply:
Ah, good catch. Thanks. (The naming conventions and layout for these docs is really confusing and irritating.)

Reply:
Anyway, the latest patch should fix this, and add docs for what that should have said SAVEGENERICSV(). (Like I said the naming convention for that is sucky.)